packages/cli/package.json

```diff
@@ -55,6 +55,7 @@
     "param-case": "3.0.4",
     "pascalcase": "1.0.0",
     "pluralize": "8.0.0",
+    "portfinder": "^1.0.32",
```

Comment:
@Josh-Walker-GM if you pin this (just remove the caret), CI will start running. There's a check early in CI that demands that all packages are pinned to avoid things from changing from underneath us as much as possible.

Reply:
Thanks @jtoar! I learned this through trial and error on a different PR 😆 ~~When I manage to make some progress on the core issue surrounding this PR I'll be sure to pin the portfinder version.~~ Thanks for pushing the pin and dedupe.

Reply:
Great and no worries!